test_engine.py

```diff
@@ -0,0 +1,372 @@
+def get_transforms(engine):
+    """
+    Description: Sets the pre-processing, and post-processing transforms for the engine.
+
+    Parameters: engine(object) : An instance of the Engine class.
+
+    Returns: None
+    """
+    engine.train_transform =Compose(
+                    [
+                        LoadImageD(Keys.all(), allow_missing_keys = True),
+                        EnsureChannelFirstD(Keys.all(), allow_missing_keys = True),
+                        ResizeD(
+                            Keys.all(), 
+                            spatial_size=[256, 256,128], 
+                            mode=("trilinear", "nearest", "nearest"), 
+                            allow_missing_keys = True
+                            ),
+                        RandZoomd(
+                            Keys.all(),
+                            prob = 0.5, 
+                            min_zoom = 0.8, 
+                            max_zoom = 1.2, 
+                            allow_missing_keys = True
+                            ),
+                        RandFlipd(
+                            Keys.all(),
+                            prob = 0.5, 
+                            spatial_axis = 1, 
+                            allow_missing_keys = True
+                            ),
+                        RandRotated(
+                            Keys.all(),
+                            range_x = 1.5, 
+                            range_y = 0, 
+                            range_z = 0, 
+                            prob = 0.5, 
+                            allow_missing_keys = True
+                            ),
+                        RandAdjustContrastd(Keys.IMAGE, prob = 0.25),
+                        NormalizeIntensityD(Keys.IMAGE, channel_wise = True),
+                        ForegroundMaskD(Keys.LABEL, threshold = 0.5, invert = True),
+                        ToTensorD(Keys.all(), allow_missing_keys = True),
+                    ]
+                )
+    
+    engine.test_transform = Compose(
+                [
+                    LoadImageD(Keys.all(), allow_missing_keys = True),
+                    EnsureChannelFirstD(Keys.all(), allow_missing_keys = True),
+                    ResizeD(
+                        Keys.all(),
+                        spatial_size=[256, 256,128], 
+                        mode=("trilinear", "nearest", "nearest"), 
+                        allow_missing_keys = True,
+                    ),
+                    NormalizeIntensityD(Keys.IMAGE, channel_wise = True),
+                    ForegroundMaskD(
+                        Keys.LABEL,
+                        threshold = 0.5, 
+                        invert = True, 
+                        allow_missing_keys = True
+                    ),
+                    ToTensorD(Keys.all(), allow_missing_keys = True),
+                ]
+            )
+    engine.postprocessing_transforms = Compose([
```

Comment:
stick to the indentation and spacing convention

Reply:
done